### PR TITLE
Update action workflows on PR image generation

### PR DESCRIPTION
**Pull request by Vincent056**
Update the action to build image for each PR, so image can be built even when PR is created from a fork.

### CODE REVIEW

**Comment by Vincent056** [comment]
this needs https://github.com/metal-toolbox/container-push/pull/68 to work

File: .github/workflows/release-pr.yml [hunk 1]
```diff
@@ -58,76 +83,26 @@ jobs:
...
-
-      - name: Get container info
-        id: container_info
-        run: |
-          image_tags="${{ needs.get-pr-number.outputs.pr-number }}"
-          echo "::set-output name=image-digest::${{ steps.docker_build.outputs.digest }}"
-          echo "::set-output name=image-tags::${image_tags}"
+      id-token: write
+      packages: write
+    uses: metal-toolbox/container-push/.github/workflows/container-push.yml@main
```
Comment (yuumasato) [inline .github/workflows/release-pr.yml:96]: Didn't we run into issues with `metal-toolbox/container-push` and `workflow_run`?
https://github.com/ComplianceAsCode/content/pull/11533

Comment (Vincent056) [reply .github/workflows/release-pr.yml:96]: I had a fix in https://github.com/metal-toolbox/container-push/pull/68

**Review by rhmdnd — APPROVED** [review]
Dependent patch looks good. Let's see how this works once it's merged.

**Pull request merged**